python/lsst/summit/utils/blockUtils.py

```diff
@@ -236,6 +236,15 @@ def augmentData(self):
         but is also much harder to maintain/debug, as the vectorized regexes
         are hard to work with, and to know which row is causing problems.
         """
+        if 'lastCheckpoint' not in self.data.columns:
+            self.log.warning("No lastCheckpoint column in data, can't parse block data")
```

Comment:
I would suggest `'lastCheckpoint'` to make it explicit that that is the name of the column and not a description, and state that empty columns are being added. Also, is it possible to print any sort of informational message about the data that was being attempted, such as an identifier or just the number of rows?

Reply:
Done. (When this happens there are almost always warnings about how little data is being found for the TMA as well btw)